index.html

```diff
@@ -652,26 +652,26 @@ <h3>
         </h3>
         <p>
           To <dfn data-lt="prompt the user to choose|prompting the user to choose" class=
-          "export">prompt the user to choose</dfn> one of several |options| associated with a
-          |descriptor|, the user agent must perform the following steps. This algorithm returns
-          either {{PermissionState/"denied"}} or one of the options.
+          "export">prompt the user to choose</dfn> one or more of several |options| associated with
+          a |descriptor|, the user agent must perform the following steps. This algorithm returns
+          either {{PermissionState/"denied"}} or the selected option(s).
         </p>
         <ol class="algorithm">
           <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"denied"}}, return
           {{PermissionState/"denied"}} and abort these steps.
           </li>
           <li>If |descriptor|'s <a>permission state</a> is {{PermissionState/"granted"}}, the user
-          agent may return one of |options| and abort these steps. If the user agent returns
-          without prompting, then subsequent <a data-lt="prompt the user to choose">prompts for the
-          user to choose</a> from the same set of options with the same |descriptor| must return
-          the same option, unless the user agent receives <a>new information about the user's
-          intent</a>.
+          agent may return one or more of |options| and abort these steps. If the user agent
+          returns without prompting, then subsequent <a data-lt="prompt the user to choose">prompts
+          for the user to choose</a> from the same set of options with the same |descriptor| must
+          return the same option(s), unless the user agent receives <a>new information about the
+          user's intent</a>.
           </li>
-          <li>Ask the user to choose one of the options or deny permission, and wait for them to
-          choose. If the calling algorithm specified extra information to include in the prompt,
-          include it.
+          <li>Ask the user to choose one or more of the options or deny permission, and wait for
```

Comment:
Then something like: 
```suggestion
          <li>Ask the user to choose one or more of the options or deny permission. If |restrictions| was passed, then restrict how many |options| to that number. Wait for
```